tools/lsp-wake/lsp.cpp

```diff
@@ -64,9 +64,6 @@ static const char *ServerNotInitialized = "-32002";
 
 DiagnosticReporter *reporter;
 
-const char *term_colour(int _) { return ""; }
-const char *term_normal() { return ""; }
-
```

Comment:
I feel somewhat worried about this change. No idea why these were stubbed out in lsp.cpp but now that it is probably depending on util/term.h these were defined twice and needed to be deleted